### DISCUSSION TITLE
How to inspect the generated config?

### DISCUSSION THREAD

**Discussion by Julien76** [nix-community/nixvim#1993] [Q&A] [answered]
Hi,
How to inspect the generated config made by nixvim?
I would like to troubleshoot an issue with pyright and seeing the lua file created by nixvim will help.

**Comment by GaetanLepage** [comment, marked as answer]
This can be done very conveniently by running the `nixvim-print-init` command shipped natively by nixvim.

**Reply by Julien76** [reply]
Awesome, exactly what I was looking for. Thanks.
